### PR TITLE
Create ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by janhommes

Ticket: [MTM-59391](https://cumulocity.atlassian.net/browse/MTM-59391)
Original PR: [6667](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/6667)

[MTM-59391]: https://cumulocity.atlassian.net/browse/MTM-59391?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: adding filtering for alarm types in the alarm overview (#6667)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md]: The Claude AI suggested to use the following title:
```suggestion
title: Add filtering for alarm types in alarm overview
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59391
+version: 1021.0.0
+---
+adding filtering for alarm types in the alarm overview (#6667)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md]: The Claude AI suggested to use the following body:
```suggestion
To improve the usability of the alarm overview, filtering for specific alarm types has been added. Users can now select which alarm types they want to see in the overview by using the new filter options. This allows users to focus on the alarms that are most relevant to them, making it easier to identify and address critical issues in a timely manner.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59391
+version: 1021.0.0
+---
+To improve the usability of the alarm overview, filtering for specific alarm types has been added. Users can now select which alarm types they want to see in the overview by using the new filter options. This allows users to focus on the alarms that are most relevant to them, making it easier to identify and address critical issues in a timely manner.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md]:
```suggestion
To improve the usability of the alarms overview, filtering for specific alarm types has been added. Users can now select which alarm types they want to see in the overview by using the new filter options. This allows users to focus on the alarms that are most relevant to them, making it easier to identify and address critical issues in a timely manner.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Add filtering for alarm types in alarm overview
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667.md]:
```suggestion
title: Filtering for alarm types added in the alarms overview
```

**Review by BeateRixen — APPROVED** [review]

**Review by janhommes — APPROVED** [review]

**Pull request merged**